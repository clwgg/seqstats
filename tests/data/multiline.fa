>seq1
ACGT
ACGT
>seq2
TGCA
