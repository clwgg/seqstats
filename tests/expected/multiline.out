Total n:	2
Total seq:	12 bp
Avg. seq:	6.00 bp
Median seq:	6.00 bp
N 50:		8 bp
Min seq:	4 bp
Max seq:	8 bp
